keyboards/cest73/tkm/tkm.h

```diff
@@ -0,0 +1,66 @@
+ * ├───┼───┼───┼───┤ ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┤ ├───┤
+ * │ + │ 9 │ 8 │ 7 │ │ Tab │ Q │ W │ E │ R │ T │ Y │ U │ I │ O │ P │ [ │ ] │  \  │ │Del│
+ * ├───┼───┼───┼───┤ ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬────│ ├───┤
+ * │ = │ 6 │ 5 │ 4 │ │ Caps │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ; │ ' │Etl│ Ent│ │PgD│
+ * ├───┼───┼───┼───┤ ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴┬───┘ ├───┤
+ * │Tab│ 3 │ 2 │ 1 │ │Shft│Shl│ Z │ X │ C │ V │ B │ N │ M │ , │ . │ / │ Shift│┌───┐│End│
+ * ├───┼───┼───┼───┤ ├────┼───┼───┴┬──┴───┴───┴───┴───┴───┴─┬─┴─┬─┴─┬─┴──┬───┘│ ↑ │└───┘
+ * │Ent│ . │ , │ 0 │ │Ctrl│GUI│Alt │                        │Alt│Mnu│Ctrl│┌───┼───┼───┐
+ * └───┴───┴───┴───┘ └────┴───┴────┴────────────────────────┴───┴───┴────┘│ ← │ ↓ │ → │
+ *                                                                        └───┴───┴───┘
+ */
+// due to a nasty PCB layout bug the R1.03 board has "6" and "7" keyboard coulmns swapped...
+
+#define LAYOUT_all( \
+	K_MO0,  K_PSCR, K_SLCK, K_PAUS, K_ESC,  K_F1,  K_F2,  K_F3,  K_F4,  K_F5,  K_F6,  K_F7,  K_F8,   K_F9,   K_F10,  K_F11,  K_F12,           K_HOME, K_PGUP, \
+	K_PMNS, K_PAST, K_PSLS, K_NLCK, K_GRV,  K_1,    K_2,   K_3,   K_4,   K_5,   K_6,   K_7,   K_8,   K_9,    K_0,    K_MINS, K_EQL, K_BSPL, K_BSPC,   K_INS,  \
+	K_PPLS, K_P9,   K_P8,   K_P7,   K_TAB,  K_Q,    K_W,   K_E,   K_R,   K_T,   K_Y,   K_U,   K_I,   K_O,    K_P,    K_LBRC, K_RBRC,      K_BSLS,     K_DEL,  \
+	K_PEQ,  K_P6,   K_P5,   K_P4,   K_CAPS, K_A,    K_S,   K_D,   K_F,   K_G,   K_H,   K_J,   K_K,   K_L,    K_SCLN, K_QUOT, K_ENTL,      K_ENT,      K_PGDN, \
+	K_PTAB, K_P3,   K_P2,   K_P1,   K_LSFT, K_NUBS, K_Z,   K_X,   K_C,   K_V,   K_B,   K_N,   K_M,   K_COMM, K_DOT,  K_SLSH, K_RSFT,          K_UP,   K_END,  \
+	K_PENT, K_PDOT, K_PMM,  K_P0,   K_LCTL, K_LGUI, K_LALT,                     K_SPC,                       K_RALT, K_APP,  K_RCTL,  K_LEFT, K_DOWN, K_RGHT  \
+	) \
+	{ \
+	{  K_MO0, K_SLCK,  K_ESC,   K_F2,   K_F4,  K_F7,   K_F8,  K_F10,  K_F12, K_PGUP }, \
+	{ K_PSCR, K_PAUS,   K_F1,   K_F3,   K_F5,  K_F6,   K_F9,  K_F11, K_HOME, K_BSPL }, \
+	{ K_PMNS, K_PSLS,  K_GRV,    K_2,    K_4,   K_7,    K_8,    K_0,  K_EQL, K_INS  }, \
+	{ K_PAST, K_NLCK,    K_1,    K_3,    K_5,   K_6,    K_9, K_MINS, K_BSPC, K_PTAB }, \
+	{ K_PPLS,   K_P8,  K_TAB,    K_W,    K_R,   K_U,    K_I,    K_P, K_RBRC, K_DEL  }, \
+	{   K_P9,   K_P7,    K_Q,    K_E,    K_T,   K_Y,    K_O, K_LBRC, K_BSLS, K_PMM  }, \
+	{  K_PEQ,   K_P5, K_CAPS,    K_S,    K_F,   K_J,    K_K, K_SCLN, K_ENTL, K_PGDN }, \
+	{   K_P6,   K_P4,    K_A,    K_D,    K_G,   K_H,    K_L, K_QUOT,  K_ENT, K_RGHT }, \
+	{ K_PENT,   K_P2, K_LSFT,    K_Z,    K_C,   K_N,    K_M,  K_DOT, K_RSFT, K_END  }, \
+	{   K_P3,   K_P1, K_NUBS,    K_X,    K_V,   K_B, K_COMM, K_SLSH,   K_UP, K_DOWN }, \
+	{ K_PDOT,   K_P0, K_LCTL, K_LGUI, K_LALT, K_SPC, K_RALT,  K_APP, K_RCTL, K_LEFT } \
+}
```

Comment:
```suggestion
#define LAYOUT_all( \
    k00, k10, k01, k11,    k02,      k12, k03, k13, k04, k14, k15, k05, k06, k16, k07, k17, k08,    k18, k09, \
\
    k20, k30, k21, k31,    k22, k32, k23, k33, k24, k34, k35, k25, k26, k36, k27, k37, k28, k19, k38,    k29, \
    k40, k50, k41, k51,    k42, k52, k43, k53, k44, k54, k55, k45, k46, k56, k47, k57, k48, k58,         k49, \
    k60, k70, k61, k71,    k62, k72, k63, k73, k64, k74, k75, k65, k66, k76, k67, k77, k68, k78,         k69, \
    k39, k90, k81, k91,    k82, k92, k83, k93, k84, k94, k95, k85, k86, k96, k87, k97, k88,       k98,   k89, \
    k80, kA0, k59, kA1,    kA2, kA3, kA4,                kA5,                kA6, kA7, kA8,  kA9, k99, k79 \
) { \
    { k00, k01, k02, k03, k04, k05, k06, k07, k08, k09 }, \
    { k10, k11, k12, k13, k14, k15, k16, k17, k18, k19 }, \
    { k20, k21, k22, k23, k24, k25, k26, k27, k28, k29 }, \
    { k30, k31, k32, k33, k34, k35, k36, k37, k38, k39 }, \
    { k40, k41, k42, k43, k44, k45, k46, k47, k48, k49 }, \
    { k50, k51, k52, k53, k54, k55, k56, k57, k58, k59 }, \
    { k60, k61, k62, k63, k64, k65, k66, k67, k68, k69 }, \
    { k70, k71, k72, k73, k74, k75, k76, k77, k78, k79 }, \
    { k80, k81, k82, k83, k84, k85, k86, k87, k88, k89 }, \
    { k90, k91, k92, k93, k94, k95, k96, k97, k98, k99 }, \
    { kA0, kA1, kA2, kA3, kA4, kA5, kA6, kA7, kA8, kA9 } \
}
```

Reply:
Now, while i agree you know your code and know how you intended it to work i have to ask what's the point in changing a portion of code that works against a more uniform one that's impossible to troubleshoot or work on later on?
Like, how in the hell am i to guess how to adjust the "6" and "7" rows back once i make the next batch of the now fixed PCB?
Also your proposed code breaks the fix too; Is that intended?

Reply:
Layout macros shouldn't say or assume anything about key assignments; they only exist to map the physical layout and ordering to the eletrical matrix array. It could also be confusing for users thinking that keys are assigned here instead of the keymap.c, which would technically work, if they used the `KC_` prefixed keycodes, but it's of course not the right way to create a keymap.

I redid this suggestion more carefully and got the exact same result, so I don't know how it's broken for you. All I'm doing is renaming the macro parameters in both places, one at a time. I'm not rearranging anything except the whitespace.

Reply:
Thanks for the clarification